doc/sphinx/language/gallina-extensions.rst

```diff
@@ -100,19 +100,22 @@ to be all present if the missing ones can be inferred or prompted for
        Rat_bottom_cond := O_S 1;
        Rat_irred_cond := one_two_irred |}.
 
-This syntax can be disabled globally for printing by
+This syntax can be disabled globally for display by
 
 .. cmd:: Unset Printing Records
```

Comment:
Following up on Hugo's remark, it would seem that `Printing Records` (with an "s") is a flag whereas `Printing Record` (without an "s") is a table. @herbelin Is that correct? @jfehrle Can you fix this?

Reply:
> it would seem that Printing Records (with an "s") is a flag whereas Printing Record (without an "s") is a table.

Confusingly similar, I'm sure others have gotten confused about this, too.